shell/platform/darwin/ios/framework/Source/accessibility_bridge.mm

```diff
@@ -206,14 +206,16 @@ void PostAccessibilityNotification(UIAccessibilityNotifications notification,
   [objects_ removeObjectsForKeys:doomed_uids];
 
   layoutChanged = layoutChanged || [doomed_uids count] > 0;
-  // We should send out only one notification per semantics update.
```

Comment:
I added this comment because we used to send node with the screen change instead of the string, so that meant sending multiple notification with different node to focus does not make sense.

But now we send string, so we still need to decide which node we should focus if we have a strong opinion.

Reply:
This also apply to  UIAccessibilityPageScrolledNotification. For now, we can't produce the scroll message and we just send node to focus. Once we do support scrolling message, we will need to send out the UIAccessibilityPageScrolledNotification even if there is layout change or route change